docs/Guide/application-commands/application-command-registry/registering-context-commands.mdx

```diff
@@ -3,35 +3,34 @@ sidebar_position: 4
 title: Registering Context Commands
 ---
 
-On the page [Registering Chat Input Commands][rcic] we have learned how to register a Chat Input Command and how to
-assign options to those commands. However there is another another type of command, Context Menu Commands.
+Context Menu Commands are another kind of application command. They show up, as the name implies, in the context menus
+of either messages or users when you right click them.
 
-Context Menu Commands show up as the name implies, in the context menu. The specific menus are those when right clicking
-a user, and those when right clicking a message.
+| Message Context Menu                    | User Context Menu                    |
+| --------------------------------------- | ------------------------------------ |
+| ![](./message-context-menu-command.png) | ![](./user-context-menu-command.png) |
 
-Message Context Menu Commands look like this:
-
-![Message Context Menu Command Example](./message-context-menu-command.png)
+## How to register a Message Context Menu Command
 
-And User Context Menu Commands look like this:
+:::warning
 
-![User Context Menu Command Example](./user-context-menu-command.png)
+There's a lot of overlap with the previous [Registering Chat Input Commands][registering-chat-input-commands] section,
+including important information that isn't repeated here, so please skim that first!
 
-## How to register a Message Context Menu Command
+:::
 
-The easiest way to register a Message Context Menu Command is to implement the [`registerApplicationCommands`][rac]
-method on a [`Command`][command] class. In that method the first parameter is the [`ApplicationCommandRegistry`][acr]
-(commonly called `registry` in code) on which you can call [`registerContextMenuCommand`][rcmc]. This looks like:
+Register a Message Context Menu Command by calling the [`registerContextMenuCommand`][rcmc] method on the registry with
+the type specified in `.setType()`, this looks like:
 
-```typescript ts2esm2cjs|{8,11-17}|{8,11-17}
+```typescript ts2esm2cjs|{8,13-17}|{8,13-17}
 import { Command } from '@sapphire/framework';
 import { ApplicationCommandType } from 'discord-api-types/v9';
 
 export class SlashCommand extends Command {
   public constructor(context: Command.Context, options: Command.Options) {
     super(context, {
       ...options,
-      description: 'Delete message and ban author.'
+      description: 'Delete message and ban author.' // In this case, this is not used
```

Comment:
```suggestion
      description: 'Delete message and ban author.'
```

It might still be for help commands (granted I am of the opinion users shouldn't have `/help` but to each their own)